config/test.js

```diff
@@ -3,18 +3,19 @@ const tsFormat = () => (new Date()).toLocaleTimeString();
 
 module.exports = {
     //test environment configuration
-	logger: {
-		colorize: true,
-		level: 'warn',
-		levels: {
-			'debug': 0,
-			'info': 1,
-			'warn': 2,
-			'error': 3
-		},
-		timestamp: tsFormat
-	},
-	database: require('../knexfile').test,
-	testsecret: 'testsecret',
-	resourceRoot: process.env.RESOURCEROOT
+    logger: {
+        colorize: true,
+        level: 'warn',
+        levels: {
+            'debug': 0,
+            'info': 1,
+            'warn': 2,
+            'error': 3
+        },
+        timestamp: tsFormat
+    },
+    database: require('../knexfile').test,
+    testsecret: 'testsecret',
+    resourceRoot: process.env.RESOURCEROOT,
+    insensitiveLikeOperator: 'LIKE'
```

Comment:
Is this because sqlite doesn't support ILIKE?

Reply:
yup and LIKE is default case insensitive.